src/bci_build/package/samba-server/README.md.j2

```diff
@@ -4,11 +4,11 @@
 
 ## Description
 
-Samba is a feature-rich Open Source implementation of the SMB and Active Directory
+Samba is a feature-rich open source implementation of the SMB and Active Directory
```

Comment:
```suggestion
Samba is a feature-rich open-source implementation of the SMB and Active Directory
```

Reply:
no, "open source" is correct. see https://documentation.suse.com/style/current/html/style-guide-db/app-terminology.html